packages/php-psr/build.sh

```diff
@@ -0,0 +1,18 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/jbboehr/php-psr
+TERMUX_PKG_DESCRIPTION="PHP extension providing the accepted PSR interfaces"
+TERMUX_PKG_LICENSE="Simplified BSD"
```

Comment:
`BSD Simplified` and not `Simplified BSD`

Reply:
Just corrected it.

Reply:
The package seems to be `BSD Simplified` and not `BSD`.

`BSD Simplified` license is available